scene/gui/text_edit.cpp

```diff
@@ -5703,7 +5730,6 @@ Vector<String> TextEdit::get_line_wrapped_text(int p_line) const {
 /* Viewport */
 // Scrolling.
 void TextEdit::set_smooth_scroll_enabled(const bool p_enabled) {
-	v_scroll->set_smooth_scroll_enabled(p_enabled);
```

Comment:
Why was this removed? This stops smooth scrolling from working when hovering over the scrollbar.

Reply:
ach, one of the things i missed when i split this PR from a bigger commit